src/test/java/fi/otavanopisto/kuntaapi/test/server/unit/tasks/AbstractTaskTest.java

```diff
@@ -0,0 +1,53 @@
+package fi.otavanopisto.kuntaapi.test.server.unit.tasks;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.fail;
+
+import java.lang.reflect.Modifier;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.Test;
+import org.reflections.Reflections;
+
+import fi.otavanopisto.kuntaapi.server.tasks.AbstractTask;
+
+public class AbstractTaskTest {
+
+  @Test
+  public void testTestUniqueHashSeeds() {
+    Reflections reflections = new Reflections("fi.otavanopisto.kuntaapi.server");
+    
+    List<AbstractTask> abstractTasks = new ArrayList<>();
+    
+    for (Class<? extends AbstractTask> abstractTypeClass : reflections.getSubTypesOf(AbstractTask.class)) {
+      try {
+        if (!Modifier.isAbstract(abstractTypeClass.getModifiers())) {
+          abstractTasks.add(abstractTypeClass.newInstance());
+        }
+      } catch (InstantiationException | IllegalAccessException e) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1166)